views/eApostilles/useStandardService.ejs

```diff
@@ -0,0 +1,28 @@
+<% pageTitle="You cannot use this service" %>
```

Comment:
When you hit this page, you loose the 'logged in' breadcrumb links for Applications, Account, Addresses and Sign out

Reply:
I'm getting an error now when hitting this page. I think its because user_data is not being passed in to determine if the user is logged in or not.
![Uploading Screenshot 2021-10-05 at 10.04.13.png…]()

Reply:
Not sure if you saw the above comment @RichardBray